boring/src/error.rs

```diff
@@ -60,16 +60,16 @@ impl ErrorStack {
 impl fmt::Display for ErrorStack {
     fn fmt(&self, fmt: &mut fmt::Formatter) -> fmt::Result {
         if self.0.is_empty() {
-            return fmt.write_str("OpenSSL error");
+            return fmt.write_str("unknown OpenSSL error");
```

Comment:
"unknown BoringSSL error"

Reply:
I changed it to BoringSSL in both places.